src/webhooks/models/webhooksFormatter.js

```diff
@@ -42,6 +44,40 @@ function slackWebhookFormat(message, options) {
     };
 }
 
+function getThresholdTeamsSummary(state, { testName, score }) {
+    let icon = '&#x1F680;';
+    if (state === WEBHOOK_EVENT_TYPE_BENCHMARK_FAILED) {
+        icon = '&#x1F622;';
+    }
+    return `${icon} *Test ${testName} got a score of ${score.toFixed(1)}`;
+}
+
+function getThresholdTeamsSubtitle(state, { benchmarkThreshold, lastScores, aggregatedReport, score }) {
+    let resultText = 'above';
+    if (state === WEBHOOK_EVENT_TYPE_BENCHMARK_FAILED) {
+        resultText = 'below';
+    }
+    return `this is ${resultText} the threshold of ${benchmarkThreshold}. ${lastScores.length > 0 ? `last 3 scores are: ${lastScores.join()}` : 'no last score to show'}` +
+        `.*\n${statsFormatter.getStatsFormatted('aggregate', aggregatedReport, { score })}\n`;
+}
+
+function teamsWebhookFormat(title, subtitle, options) {
+    return {
+        '@type': 'MessageCard',
+        '@context': 'http://schema.org/extensions',
+        'summary': 'Predator Notification',
+        themeColor: WEBHOOK_TEAMS_DEFAULT_THEME_COLOR,
+        sections: [
+            {
+                activityTitle: `![Mikey](https://assets.website-files.com/5ceb9d8c5d0f4725dca04998/5cf3af2d6e00f5d87f38869e_mickey-clean.png)${title}`,
+                text: `${subtitle}`,
+                activityImage: 'https://assets.website-files.com/5ceb9d8c5d0f4725dca04998/5cf3af2d6e00f5d87f38869e_mickey-clean.png',
```

Comment:
@enudler @NivLipetz 
Can we assume that this CDN is highly available?
Should we host these files on our server?

what do you think?

Reply:
It is your website, not mine.  😸 
No longer an issue, because I have simplified the teams messages to just use text and emojis, to bring them in line with the current Slack messages.
Fixed in commit 6719dd2a4a5e2d612b2793d2536f54afec2b6477